### PR TITLE
Feature/benchmarking

### PR DESCRIPTION
**Pull request by NickvonWit**
i am sorry for the formatting, i guess i have an older version of it:/

### CODE REVIEW

**Review by casparschucan — APPROVED** [review]
lgtm

File: test/orthotree/OrthoTreeBenchmarking.cpp
Comment (mazzeaar) [inline test/orthotree/OrthoTreeBenchmarking.cpp:1]: merge dphpc_octree (includes new particle distributions, same ones as in the paper)

Comment (mazzeaar) [reply test/orthotree/OrthoTreeBenchmarking.cpp:1]: ah nvm, just copy the distributions

**Review by mazzeaar — APPROVED** [review]
nice! add the new distributions and then merge

File: test/orthotree/OrthoTreeBenchmarking.cpp
Comment (NickvonWit) [reply test/orthotree/OrthoTreeBenchmarking.cpp:1]: i copied them

**Pull request merged**